src/tree/box.js

```diff
@@ -101,6 +101,18 @@ export default function box({
         selections.select(node, selectionObj);
       }
     })
+    .each((node, index, cards) => {
+      if (!interactions.isIE) {
+        Touche(cards[index]).tap({
+          end: () => {
+            if (!wrapperState.constraints.active && node.data.id !== 'Root') {
```

Comment:
Pretty sure we don't have to check for `Root` anymore, since it is an empty div I think